packages/0x.js/package.json

```diff
@@ -61,7 +61,7 @@
     "copyfiles": "^1.2.0",
     "coveralls": "^3.0.0",
     "dirty-chai": "^2.0.1",
-    "ethereumjs-testrpc": "4.0.1",
+    "ethereumjs-testrpc": "^6.0.3",
```

Comment:
How about we remove the `^`. Don't trust them to adhere to semantic versioning yet.